src/github-api-helper.ts

```diff
@@ -84,11 +84,11 @@ export async function getDefaultBranch(
 ): Promise<string> {
   return await retryHelper.execute(async () => {
     core.info('Retrieving the default branch name')
-    const octokit = getOctokit(authToken, {baseUrl: baseUrl})
+    const octokit = github.getOctokit(authToken, {baseUrl: baseUrl})
```

Comment:
The `baseUrl` must be converted to `serverApiUrl`.

I fixed this by https://github.com/actions/checkout/pull/1286.

Reply:
Oh thanks!